nav2_behavior_tree/nav2_tree_nodes.xml

```diff
@@ -16,7 +16,19 @@
     <Action ID="CancelControl">
       <input_port name="service_name">Service name to cancel the controller server</input_port>
     </Action>
-    
+
+    <Action ID="CancelBackUp">
+      <input_port name="service_name">Service name to cancel the backup recovery</input_port>
```

Comment:
Is there also not `server_timeout` as your PR in documentation shows?

@padhupradheep please, please slow down and think through these requested changes. Don't just blitz through them. For every 10 of these easily overlookable errors I find, there could easily be 1-2 that slip through. I appreciate your help in Nav2, but please go slow, be methodical, make sure things are complete and correct.

Reply:
Its better to get 1 thing done, and done well, than 5 things sent but then we spend 2 months on getting it to be merged :+1: :wink:

Reply:
I agree on the point. I actually missed it in the cancelcontrol BT as well. That got me a bit off guard. Sorry for the trouble.

Reply:
No worries, not trying to scold, just letting you know I think you can do better :wink: Attention to detail is the game of engineering. I was just like you for a number of years until I started leading projects and having my own hastfulness bite me in the butt. It has made me more cautious and thoughtful - I think you'll find that too over time as you work on the same systems year after year. Old decisions you made come back to haunt you :laughing:

Reply:
No issues, I really understand that. I’m happy to get some honest feedback on the fly from you, which really helps me develop as a engineer. But thanks for being patient. Let me see what I missed and add it ✌️